### PR TITLE
Preserve data

### PR DESCRIPTION
**Pull request by moise-mulungu**
On this milestone, We:
- Saved data for `@books,` `@people` and `@rentals`.
- Created a new file named `data.rb` for data methods.
- Added loaded methods inside the data file.
- Created a `saved.rb` file to save all the data.
- Updated the README file.

### CODE REVIEW

File: data.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
+require './student'
+require './teacher'
+require './rental'
+
+def files
+  File.write('./json/book.json', []) unless File.exist?('./json/book.json')
+  File.write('./json/person.json', []) unless File.exist?('./json/person.json')
+  File.write('./json/rental.json', []) unless File.exist?('./json/rental.json')
+end
```
Comment (omar-muhamad) [inline data.rb]: - [ ] When the user adds books or people and exits the app this data suppose to be saved in the `.json` meant for that but this is not happening, So please fix that.

File: app.rb [hunk 2]
```diff
@@ -3,11 +3,13 @@
...
+require './data'
 
 class Library
   def initialize
-    @books = []
-    @people = []
+    files
+    @books = books_loaded
+    @people = persons_loaded
     @rentals = []
```
Comment (omar-muhamad) [inline app.rb]: - [ ] Please add `rentals_loaded` that you created in `data.rb` here instead of an empty array to load rentals saved when the app loaded.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @moise-mulungu and @ciraganenicole,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- Nothing to mention.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by uwadonat — APPROVED** [review]
Hi **@Team,**

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# Hightlights
- No linter errors ✔️
- Clean and well organized code ✔️
appr3

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**